provisionerd/provisionerd.go

```diff
@@ -290,7 +291,7 @@ func (p *Server) acquireLoop() {
 	defer p.wg.Done()
 	defer func() { close(p.acquireDoneCh) }()
 	ctx := p.closeContext
-	for {
+	for retrier := retry.New(10*time.Millisecond, 1*time.Second); retrier.Wait(ctx); {
```

Comment:
NOTE: This is causing some flakiness currently, because jobs are timing out waiting for provisioners, now that we're introducing some backoff.

Looking into it...